.github/workflows/tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - name: Install 3rd party from apt
         run: |
-            sudo apt install unar zlib1g-dev liblzo2-dev
+            sudo apt install unar zlib1g-dev liblzo2-dev lzop
```

Comment:
It doesn't matter, does it? I would not add any of the non-required dependencies to checks, it just takes longer to set them up. `zlib1g-dev` `liblzo2-dev` were needed only because pyright couldn't find dependencies.
Anyway, we should solve this in a different issue/PR., not here.